docs/devel/api_changes.md

```diff
@@ -236,7 +236,8 @@ assumptions.  If you have added any fields which need very careful formatting
 "this slice will always have at least 1 element", you may get an error or even
 a panic from the `serialization_test`.  If so, look at the diff it produces (or
 the backtrace in case of a panic) and figure out what you forgot.  Encode that
-into the fuzzer's custom fuzz functions.
+into the fuzzer's custom fuzz functions.  Hint: if you added defaults for fields,
+those fields will need to be fuzzed.
```

Comment:
s/need to be fuzzed/need to be custom-handled/

Probably point to an example in /pkg/api/testing/fuzzer.go : eg. https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/testing/fuzzer.go#L232